src/main/java/com/glencoesoftware/bioformats2raw/Converter.java

```diff
@@ -1506,33 +1506,63 @@ private void setSeriesLevelMetadata(int series, int resolutions)
     multiscale.put("metadata", metadata);
     multiscale.put("version", nested ? "0.2" : "0.1");
     multiscales.add(multiscale);
-    List<Map<String, String>> datasets = new ArrayList<Map<String, String>>();
+
+    IFormatReader v = null;
+    IMetadata meta = null;
+    String axisOrder = null;
+    try {
+      v = readers.take();
+      meta = (IMetadata) v.getMetadataStore();
+
+      if (dimensionOrder != null) {
+        axisOrder = dimensionOrder.toString();
+      }
+      else {
+        axisOrder = v.getDimensionOrder();
+      }
+    }
+    finally {
+      readers.put(v);
+    }
+
+    List<Map<String, Object>> datasets = new ArrayList<Map<String, Object>>();
     for (int r = 0; r < resolutions; r++) {
       resolutionString = String.format(
               scaleFormatString, getScaleFormatStringArgs(series, r));
       String lastPath = resolutionString.substring(
               resolutionString.lastIndexOf('/') + 1);
-      datasets.add(Collections.singletonMap("path", lastPath));
-    }
-    multiscale.put("datasets", datasets);
 
-    String axisOrder = null;
-    if (dimensionOrder != null) {
-      axisOrder = dimensionOrder.toString();
-    }
-    else {
-      IFormatReader reader = readers.take();
-      try {
-        axisOrder = reader.getDimensionOrder();
-      }
-      finally {
-        readers.put(reader);
+      List<Map<String, Object>> transforms =
+        new ArrayList<Map<String, Object>>();
+      Map<String, Object> scale = new HashMap<String, Object>();
+      scale.put("type", "scale");
+      List<Double> axisValues = new ArrayList<Double>();
+      double resolutionScale = Math.pow(PYRAMID_SCALE, r);
+      for (int i=axisOrder.length()-1; i>=0; i--) {
+        Quantity axisScale = getScale(meta, series, axisOrder, i);
+        if (axisScale != null) {
```

Comment:
This will need to be updated to store the following values:
- `axisScale.value().doubleValue() * resolutionScale` if downsampling happens along the dimension and there is some scaling metadata associated with the dimension
- `resolutionScale` if downsampling happens along the dimension and there is no scaling metadata associated with the dimension
- `1.0` otherwise